renovate.json

```diff
@@ -21,6 +21,12 @@
 			"groupSlug": "all-non-major",
 			"automerge": true,
 			"automergeType": "branch"
+		},
+		{
+			"groupName": "eslint/prettier/biome",
+			"automerge": false,
```

Comment:
why? if the dependency requires a PR, it will create it due to broken CI, if not, why shouldn't we just automerge?

Reply:
good callout